src/api/Nhs.Appointments.Persistance/BookingCosmosDocumentStore.cs

```diff
@@ -16,11 +16,16 @@ public BookingCosmosDocumentStore(ITypedDocumentCosmosStore<BookingDocument> boo
         _indexStore = indexStore;
     }
            
-    public Task<IEnumerable<Booking>> GetInDateRangeAsync(string site, DateTime from, DateTime to)
+    public Task<IEnumerable<Booking>> GetInDateRangeAsync(DateTime from, DateTime to, string site)
     {
         return _bookingStore.RunQueryAsync<Booking>(b => b.Site == site && b.From >= from && b.From <= to);
-    }       
-    
+    }
+
+    public Task<IEnumerable<Booking>> GetInDateRangeAsync(DateTime from, DateTime to)
+    {
+        return _bookingStore.RunQueryAsync<Booking>(b => b.From >= from && b.From <= to);
```

Comment:
This will be a cross partition query and won't be performant, we will need to consider an alternate approach such as including the date time of the appointment in the booking index and querying against that.